src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+$primary_color: #ddd;
+$secondary_color: #fff;
+$btn_border_color: red;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'montserrat', sans-serif;
+  background: transparent;
+  .slider {
+    display: none;
+  }
+}
+
+.bgimg {
+  background: url('../css/img/cv.jpg');
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  filter: blur(5px);
+}
+
+.header {
+  max-width: 1000px;
```

Comment:
Instead of setting a `max-width`, it prefers using `media queries` for a responsive design, since max-width always performs badly at legacy IEs

- https://css-tricks.com/css-media-queries/
- https://caniuse.com/#search=max-width